libcloudforensics/providers/aws/forensics.py

```diff
@@ -206,9 +206,29 @@ def StartAnalysisVm(
   Returns:
     Tuple[AWSInstance, bool]: a tuple with a virtual machine object
         and a boolean indicating if the virtual machine was created or not.
+
+  Raises:
+    RuntimeError: When multiple AMI images are returned.
   """
+
   aws_account = account.AWSAccount(
       default_availability_zone, aws_profile=dst_profile)
+
+  # If no AMI ID is given we use the default Ubuntu 18.04
+  # in the region requested.
+  if not ami:
+    qfilter = [{'Name': 'name', 'Values': [UBUNTU_1804_FILTER]}]
+    ami_list = aws_account.ListImages(qfilter)
+    # We should only get 1 AMI image back, if we get multiple we
+    # have no way of knowing which one to use.
+    if len(ami_list) > 1:
+      print('AMI images found:')
+      for image in ami_list:
+        print('Name: {0:s}, ImageId: {1:s}, Location: {2:s}'.format(
+            image['Name'], image['ImageId'], image['ImageLocation']))
+      raise RuntimeError('error - ListImages returns >1 AMI image!')
```

Comment:
Printing things and then raising sounds like two different channels by which to give feedback to the user and I don't think that's optimal. A good middleground would be to tweak the exception string to list the images 

```     
raise RuntimeError('error - ListImages returns >1 AMI image: [{0:s}]'.format(
    ', '.join([image['Name'] for image in ami_list])
)
```

Replace `image['Name']` by whatever information the caller can filter on to narrow down their image choice.

Reply:
done